frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -182,6 +185,7 @@ export const VMDisks: React.FC<VMDisksProps> = ({ vmLikeEntity, pvcs, datavolume
               setIsCreating(false);
               setCreateError(error);
             },
+            forceRerender: () => setRerenderFlag(!rerenderFlag),
```

Comment:
Shouldn't it be set to `true`?

Reply:
no the change in props needs to occur, hence flipping the flag

Reply:
I still consider it as an ugly hack ...

I had difficulty to determine real semantics of this flag when reading the code for the first time, can you please either leave a short descriptive comment or rename to sort of "forceRerenderFlag". Just for future generations ...

Reply:
sure,  it is an hack, but it is there because we have no other simple choice before the release other than to release a buggy feature

* renamed to `forceRerenderFlag`